src/index.js

```diff
@@ -159,7 +159,7 @@ function createCondition(...args) {
 }
 
 let commonReplacer = [
-  /[\0\n\r\b\t\\'"\x1a]/g, (s) => {
+  /[\0\n\r\b\t\\\x1a]/g, (s) => {
```

Comment:
bug2: no switch case for quote/doublequote/backquote. both cases will be processed later in quoteVal:
```
this.value.replace(...commonReplacer).replace(/'/g, "''")
```
So just removed from regexp